src/test/groovy/transform/stc/GenericsSTCTest.groovy

```diff
@@ -1613,6 +1613,8 @@ assert result == 'ok'
             class ComponentFixture<T extends Component> {}
             class JButtonFixture extends ComponentFixture<JButton> {}
             class ContainerFixture<T extends Container> extends ComponentFixture<T> {}
+
+            @groovy.transform.TypeChecked(groovy.transform.TypeCheckingMode.SKIP)
```

Comment:
This class wasn't type checked in the example in the issue (GROOVY-5981), so this should be ok.